README.md

```diff
@@ -1 +1,81 @@
 # Template Marlin Processor
+
+A template Marlin package with a skeleton processor. It contains the following:
+
+* A template structure for organizing your code.
+* A CMakeLists.txt for compilation.
+* A skeleton processor that you can modify.
+
+Note that official Marlin packages follow a slightly different template. This is because they are based on old CMake conventions. The template documented here has been updated for modern CMake.
+
+## Compiling
+Start by making a fork or directly cloning the template repository.
+
+```bash
+cd /scratch/${USER}/tutorial2024
```

Comment:
I would remove references specific to your tutorial.
- This template is independent of the tutorial and having paths in the instructions corresponding to it is confusing.
- I don't believe this request detracts from the tutorial, since the instructions that everyone will be following are inside the tutorial itself.

Reply:
sorry, my mistake. Definitely I agree and I removed it.